net.krafting.PedantiK.yml

```diff
@@ -0,0 +1,66 @@
+app-id: net.krafting.PedantiK
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: PedantiK.py
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+
+modules:
+  - name: OpenBLAS
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+    - -DBUILD_SHARED_LIBS=ON
+    - -DBUILD_TESTING=OFF
+    sources:
+      - type: git
+        url: https://github.com/OpenMathLib/OpenBLAS
+        tag: 'v0.3.28'
+        commit: 5ef8b1964658f9cb6a6324a06f6a1a022609b0c5
+
+  - name: meson-python
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/mesonbuild/meson-python
+        tag: '0.15.0'
+        commit: e847bdd918ba6686d9b0502e9ee6d83d5145ae84
+
+  - python3-pyproject_metadata.json
+
+  - python3-numpy.json
+
+  - python3-pythran.json
+
+  - python3-pybind11.json
+
+  - python3-gensim.json
+
+  - python3-requests.json
+
+  - python3-unidecode.json
+
+  - name: PedantiK
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 PedantiK.py /app/bin/PedantiK.py
+      - cp -r utils/ /app/bin/utils/
+      - install -Dm 0644 frWac_non_lem_no_postag_no_phrase_200_cbow_cut100.bin  /app/models/frWac_non_lem_no_postag_no_phrase_200_cbow_cut100.bin
+      - install -Dm 0644 net.krafting.PedantiK.desktop  /app/share/applications/net.krafting.PedantiK.desktop
+      - install -Dm 0644 net.krafting.PedantiK.metainfo.xml  /app/share/metainfo/net.krafting.PedantiK.metainfo.xml
+      - install -Dm 0644 images/net.krafting.PedantiK.svg /app/share/icons/hicolor/scalable/apps/net.krafting.PedantiK.svg
+    sources:
+      - type: file
+        url: https://embeddings.net/embeddings/frWac_non_lem_no_postag_no_phrase_200_cbow_cut100.bin
+        sha256: d51f3259d1f6fe12155a447623a82695c6d72916b1b61e150e2715c034b6c880
```

Comment:
what is this file ?

Reply:
This is a model file for word similarities, from here : https://fauconnier.github.io/#data